mmdet/models/dense_heads/yolo_head.py

```diff
@@ -303,7 +433,9 @@ def loss(self,
              gt_bboxes,
              gt_labels,
              img_metas,
-             gt_bboxes_ignore=None):
+             gt_bboxes_ignore=None,
+             target_maps_list=None,
```

Comment:
Suggest moving the IPU-related code into IPUYOLOV3Head which inherit from YOLOV3Head. In this way, users without IPU do not have to read the ipu related code.